src/components/Table/components/ExpandableHeader.tsx

```diff
@@ -35,6 +35,8 @@ export function ExpandableHeader<R extends Kinded>(props: ExpandableHeaderProps<
         if (isFunction(column.expandColumns)) {
           setIsLoading(true);
           await tableState.loadExpandedColumns(column);
+          // manually calling this as loadExpandedColumns does not toggle
+          tableState.toggleExpandedColumn(column.id);
           setIsLoading(false);
         } else {
           tableState.toggleExpandedColumn(column.id);
```

Comment:
**nit:** Could do something like:
```
        if (isFunction(column.expandColumns)) {
          setIsLoading(true);
          await tableState.loadExpandedColumns(column);          
          setIsLoading(false);
        }
        tableState.toggleExpandedColumn(column.id);
```
De-dupes the two `toggleExpandedColumn` calls. Instead of having one for both conditionals, now we just have one.